helixbass/tree-sitter-grep
# Use rayon + WalkParallel

In this PR:
- "hand off" from `ignore` `WalkParallel` (for enumerating project files in parallel) to `rayon` (for running the tree-sitter querying in parallel) (via its `.par_bridge()` helper)

To test:
Behavior should still work and performance still appears pretty comparable to the manual channel + thread-pool version in `dont-block-on-enumerate-project-files`

Based on `dont-block-on-enumerate-project-files` (again maybe it would be preferable to see the diff against the "base branch" `filter-plugin-c-ffi`, feel free to change it)
